### ISSUE TITLE
Requirements for more pretrained weights

### ISSUE THREAD

**Issue by Ming-er** [YuanGongND/ltu#8]
Hi, thanks for your brilliant work. I want to know if there are any plans to release more pretrained weights of LTU, such as checkpoints for stage 1, 2, 3 (so that we can skip over some trivial pretraining steps and explore more meaningful problems)?

**Comment by YuanGongND** [comment]
yes, can you make a list of them? I will try to accomendate these.

**Comment by Ming-er** [comment]
Thanks for your kindness. I think releasing the checkpoints of the training stage 1,2,3 in Tables 3 of LTU, and the checkpoints of the training stage 1,2 in Tables 3 of LTU-AS may be beneficial.

**Comment by YuanGongND** [comment]
will do, please allow some time. 

-Yuan

**Comment by YuanGongND** [comment]
Done. 

I have provided checkpoints of all stages, together with our loss log. Please see https://github.com/YuanGongND/ltu/blob/main/src/ltu/train_script/finetune_toy_low_resource.sh. Please let me know if there's any further questions.

-Yuan

**Comment by Ming-er** [comment]
Many thanks. I will check it soon.
